osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -56,6 +57,7 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
                 Mods = mods,
                 AimStrain = aimRating,
                 SpeedStrain = speedRating,
+                FlashlightStrain = flashlightRating,
```

Comment:
Just a minor complaint, but can we use something like `FlashlightDifficulty` instead? Using strain is confusing in my opinion. Ideally change aim/speed variables too but that's out of scope and I assume it can't be changed easily.

Reply:
That's reasonable, I've renamed it to `FlashlightRating` to be consistent with `StarRating` and yeah this should be done with aim and speed too I think